config/binutils/binutils.in.2

```diff
@@ -28,8 +28,8 @@ if ELF2FLT_GIT
 
 config ELF2FLT_GIT_CSET
     string
-    prompt "git cset"
-    default "9dbc458c6122c495bbdec8dc975a15c9d39e5ff2"
+    prompt "git cset or ref"
+    default "ref=HEAD"
```

Comment:
I am a bit hesitant on this, as this affects the reproducibility of the builds. I.e., if at some point a sample in crosstool-ng starts failing, you cannot easily find a good revision for 'git bisect' in case a failure is due to some change in elf2flt - even the revisions that used to build fine would now be returning failures. Given that we don't have a build archive for all samples, tracing failures would be... fun.